project_milestone/tests/test_project_milestone.py

```diff
@@ -0,0 +1,45 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestProjectMilestone(common.TransactionCase):
+
+    def test_create_data(self):
+        # Create a new project with the tests
+        test_project = self.env['project.project'].create({
+            'name': 'TestProject'})
+
+        # Add a test Milestone to the project
+        test_project_milestone_1 = self.env['project.milestone'].create({
+            'name': 'TestMilestone_1',
+            'project_id': test_project.id})
+        test_project_milestone_2 = self.env['project.milestone'].create({
+            'name': 'TestMilestone_2',
+            'project_id': test_project.id})
+
+        # Check Milestone Sequences

```

Comment:
So, once you have all created in the `setUp`, this would be a separate test that only tests the milestone sequence is applied correctly:

```suggestion
    def test_milestone_sequences(self):
        """Milestone sequences are applied correctly."""
```